### PR TITLE
Resolve issue #689 by building in package imdbphp

### PR DESCRIPTION
**Pull request by mesa57**
@Sweepr Please review and / or apply styleCI

### CODE REVIEW

File: lib/services/MediaInformation/Services_MediaInformation_Imdb.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
-
+use imdbphp\�mdb;
```
Comment (Sweepr) [inline lib/services/MediaInformation/Services_MediaInformation_Imdb.php:2]: @mesa57 
Is imdbphp\Ímdb; the correct syntax for this call?

I would expect: imdbphp\imdb;

Comment (mesa57) [reply lib/services/MediaInformation/Services_MediaInformation_Imdb.php:2]: It is indeed with a capital. That's how it's named in the package, so I cannot change that.

Comment (Sweepr) [reply lib/services/MediaInformation/Services_MediaInformation_Imdb.php:2]: Akward, it's an i with acute.

Not want to be picky or anything, but if it works.. :smile:

Comment (mesa57) [reply lib/services/MediaInformation/Services_MediaInformation_Imdb.php:2]: Ehh, it should be a capital, but it isn't here on github. Very strange. I will look into that.

**Comment by mesa57** [comment]
I will close this pull request and create a new one

**Pull request closed without merging**